src/main/java/blackjack/domain/card/Cards.java

```diff
@@ -0,0 +1,75 @@
+package blackjack.domain.card;
+
+import blackjack.domain.card.type.CardNumberType;
+import blackjack.domain.card.type.CardShapeType;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Cards {
+    private static final int BLACKJACK_SIZE = 2;
+    private static final int BLACKJACK_SCORE = 21;
+    private final List<Card> cards = new ArrayList<>();
+
+    private Cards() {
+    }
+
+    public static Cards createAllShuffledCards() {
+        Cards cards = new Cards();
+        cards.createAllCards();
+        cards.shuffle();
+        return cards;
+    }
+
+    public static Cards createEmptyCards() {
+        return new Cards();
+    }
+
+    public int getScore() {
+        cards.sort((o1, o2) -> -Integer.compare(o1.getValue(), o2.getValue()));
```

Comment:
아래 처럼 쓸 수도 있는데, `getValue` 와 `getValue(int scoreSum)` 이름이 같아서 오류가 날거에요. 둘중 하나 메소드 이름을 수정하면 좋을 것 같아요 :)
```suggestion
        cards.sort(Comparator.comparing(Card::getValue).reversed());
```

Reply:
넵 반영하겠습니다!! 😄